entities/sentry/config.go

```diff
@@ -25,19 +25,20 @@ import (
 // ConfigOpts all map to environment variables. For example:
 //   - SENTRY_ENABLED=true -> ConfigOpts.Enabled=true
 type ConfigOpts struct {
-	Enabled           bool              `json:"enabled" yaml:"enabled"`
-	DSN               string            `json:"dsn" yaml:"dsn"`
-	Debug             bool              `json:"debug" yaml:"debug"`
-	Tags              map[string]string `json:"tags" yaml:"tags"`
-	Release           string            `json:"release" yaml:"release"`
-	Environment       string            `json:"environment" yaml:"environment"`
-	TracingEnabled    bool              `json:"tracing_enabled" yaml:"tracing_enabled"`
-	ProfilingEnabled  bool              `json:"profiling_enabled" yaml:"profiling_enabled"`
-	ErrorSampleRate   float64           `json:"error_sample_rate" yaml:"error_sample_rate"`
-	TracesSampleRate  float64           `json:"traces_sample_rate" yaml:"traces_sample_rate"`
-	ProfileSampleRate float64           `json:"profile_sample_rate" yaml:"profile_sample_rate"`
-	ClusterId         string            `json:"cluster_id" yaml:"cluster_id"`
-	ClusterOwner      string            `json:"cluster_owner" yaml:"cluster_owner"`
+	Enabled                bool              `json:"enabled" yaml:"enabled"`
+	DSN                    string            `json:"dsn" yaml:"dsn"`
+	Debug                  bool              `json:"debug" yaml:"debug"`
+	Tags                   map[string]string `json:"tags" yaml:"tags"`
+	Release                string            `json:"release" yaml:"release"`
+	Environment            string            `json:"environment" yaml:"environment"`
+	ErrorReportingDisabled bool              `json:"error_reporting_disabled" yaml:"error_reporting_disabled"`
+	TracingDisabled        bool              `json:"tracing_disabled" yaml:"tracing_disabled"`
```

Comment:
hmm, why did this change from `TracingEnabled` to `TracingDisabled`, wouldn't it be cleaner to have `*Enabled` options?

Reply:
Talked offline: the idea is to enable everything by default and only have to disable option on a case by case basis if needed.